akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -227,3 +232,62 @@ This can be useful when initial contacts are provided from some kind of service
 are entirely dynamic and the entire cluster might shut down or crash, be restarted on new addresses. Since the
 client will be stopped in that case a monitoring actor can watch it and upon `Terminate` a new set of initial
 contacts can be fetched and a new cluster client started.
+
+## Migration to Akka gRPC
+
+Cluster Client is deprecated and it is not advised to build new applications with Cluster Client.
+We recommend using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html) over using Cluster Client.
+
+Cluster Client is convenient to use if you have Akka actors on both client and server side, but
+a more decoupled, and therefore better, solution would be to use Akka gRPC and define
+application specific protocol buffer messages and gRPC service calls. The benefits would be:
+
+* Easier to update clients and servers independent of each other.
```

Comment:
Nit: bullet points don't need a period at the end

Reply:
curious to learn: until one of them becomes two sentences?

Reply:
I remember it being that way for scientific writing though I dug into the minutia for technical writing from Oxford and Cambridge U, Google, Microsoft. I think it's fine if we don't go with those rules. What you have is ok.